mitosheet/mitosheet/user/upgrade.py

```diff
@@ -252,6 +262,8 @@ def try_upgrade_user_json_to_current_version() -> None:
         user_json_object = upgrade_user_json_version_8_to_9(user_json_object)
     if user_json_object[UJ_USER_JSON_VERSION] == 9:
         user_json_object = upgrade_user_json_version_9_to_10(user_json_object)
+    if user_json_object[UJ_USER_JSON_VERSION] == 10:
+        user_json_object = upgrade_user_json_version_10_to_11(user_json_object)
```

Comment:
We can test this in `test_user.py` where we have tests like `test_upgrades_version_3_to_version_4`